lib/index.js

```diff
@@ -226,6 +233,12 @@ export default {
           notifyError(e, `${this.executablePath} ${execArgs.join(' ')}`);
           return null;
         }
+
+        // handle unique spawning: killed execs will return null
+        if (result === null) {
+          return [];
```

Comment:
Return `null` here, telling Linter not to update any messages it currently has saved for the provider. (`[]` tells it there are _no_ messages, so it will wipe the slate clean!)